partner_ref_unique/models/res_company.py

```diff
@@ -12,6 +12,7 @@ class ResCompany(models.Model):
         selection=[
             ("none", "None"),
             ("companies", "Only companies"),
+            ("exclude_corporative", "All, except partners in company"),
```

Comment:
does it not make more sense like this ?
```suggestion
            ("exclude_corporative", "All, except contacts of a partner"),
```